test/commutative_tests.jl

```diff
@@ -15,26 +15,30 @@ function f_commute(du,u,p,t)
 end
 
 function f_commute_analytic(u0,p,t,W)
- tmp = (A+1.01I-(B^2))*t + B*sum(W)
+ tmp = (A+1.01I-2*(B^2))*t + B*sum(W)
```

Comment:
what's the 2 for? I forget the analytical solution derivation here. It's from one of Platen's papers.

Reply:
that is because we have 4 noise processes.
![image](https://user-images.githubusercontent.com/26359940/59534215-449c0a80-8f0b-11e9-8410-ffb6934b7d79.png)
We have a equation of this form and length of dW = 4